apps/web/pages/availability/[schedule].tsx

```diff
@@ -158,282 +86,40 @@ export default function Availability() {
-                ...values,
-              });
-          }}
-          className="flex flex-col sm:mx-0 xl:flex-row xl:space-x-6">
-          <div className="flex-1 flex-row xl:mr-0">
-            <div className="border-subtle mb-6 rounded-md border">
-              <div>
-                {typeof me.data?.weekStart === "string" && (
-                  <Schedule
-                    control={form.control}
-                    name="schedule"
-                    weekStart={
-                      ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"].indexOf(
-                        me.data?.weekStart
-                      ) as 0 | 1 | 2 | 3 | 4 | 5 | 6
-                    }
-                  />
-                )}
-              </div>
-            </div>
-            <div className="border-subtle my-6 rounded-md border">
-              {schedule?.workingHours && (
-                <DateOverride userTimeFormat={timeFormat} workingHours={schedule.workingHours} />
-              )}
-            </div>
-          </div>
-          <div className="min-w-40 col-span-3 hidden space-y-2 md:block lg:col-span-1">
-            <div className="xl:max-w-80 w-full pr-4 sm:ml-0 sm:mr-36 sm:p-0">
-              <div>
-                <Skeleton
-                  as={Label}
-                  htmlFor="timeZone-lg-viewport"
-                  className="mb-0 inline-block leading-none">
-                  {t("timezone")}
-                </Skeleton>
-                <Controller
-                  name="timeZone"
-                  render={({ field: { onChange, value } }) =>
-                    value ? (
-                      <TimezoneSelect
-                        inputId="timeZone-lg-viewport"
-                        value={value}
-                        className="focus:border-brand-default border-default mt-1 block w-72 rounded-md text-sm"
-                        onChange={(timezone) => onChange(timezone.value)}
-                      />
-                    ) : (
-                      <SelectSkeletonLoader className="mt-1 w-72" />
-                    )
-                  }
-                />
-              </div>
-              <hr className="border-subtle my-6 mr-8" />
-              <div className="rounded-md">
-                <Skeleton as="h3" className="mb-0 inline-block text-sm font-medium">
-                  {t("something_doesnt_look_right")}
-                </Skeleton>
-                <div className="mt-3 flex">
-                  <Skeleton as={Button} href="/availability/troubleshoot" color="secondary">
-                    {t("launch_troubleshooter")}
-                  </Skeleton>
-                </div>
-              </div>
-            </div>
-          </div>
-        </Form>
-      </div>
-    </Shell>
+      isDeleting={deleteMutation.isPending}
+      isLoading={isPending}
+      isSaving={updateMutation.isPending}
+      timeFormat={timeFormat}
+      weekStart="Sunday"
```

Comment:
I think this need to use the me/session data, this is how it was written before

```weekStart={
                      ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"].indexOf(
                        me.data?.weekStart
                      ) as 0 | 1 | 2 | 3 | 4 | 5 | 6
                    }```

Reply:
@ThyMinimalDev this is done internally in our atom, all we need from the outside is the `me.data?.weekStart` which is a string so I guess the only thing that I need to pass in from the outside is the weekStart from a user